src/main/java/net/minecraftforge/fml/client/GuiSortingProblem.java

```diff
@@ -24,20 +24,44 @@
 import net.minecraftforge.fml.common.toposort.ModSortingException;
 import net.minecraftforge.fml.common.toposort.ModSortingException.SortingExceptionData;
 
+/**
+ * This class represents a GUI screen that is displayed when a mod sorting cycle is detected during Minecraft's loading process.
```

Comment:
The formulation here is very strange and even completely unnecessary. Please rewrite it manually, you just need to clarify what it is?
```java
- This class represents a GUI screen
- It extends the GuiScreen class from Minecraft's client GUI system.
+ display when a mod sorting cycle is detected during Minecraft's loading process.
```